conf/caas_log_config.yaml

```diff
@@ -0,0 +1,52 @@
+#Log config file for Caffe as a Service
+
+version: 1
+
+disable_existing_loggers: False
+
+formatters:
+    simple:
+        format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+handlers:
+    console:
+        class: logging.StreamHandler
+        level: DEBUG
+        formatter: simple
+        stream: ext://sys.stdout
+
+    info_file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: INFO
+        formatter: simple
+        filename: test/caas/info.log
+        maxBytes: 10485760 # 10MB
+        backupCount: 20
+        encoding: utf8
+
+    error_file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: ERROR
+        formatter: simple
+        filename: test/caas/errors.log
+        maxBytes: 10485760 # 10MB
+        backupCount: 20
+        encoding: utf8
+
+    debug_file_handler:
+        class: logging.handlers.RotatingFileHandler
+        level: DEBUG
+        formatter: simple
+        filename: test/caas/debug.log
+        maxBytes: 10485760 # 10MB
```

Comment:
Sorry, I didn't got it. Is 'caas' a module ? If yes, 'caas' has three different log files viz info.log/error.log/debug.log. And I am assuming log.info will go to info.log, log.debug to debug.log and so on. Is that the norm ? Person debugging will have to open 3 files together and then match timestamps across these 3 files to understand what's going on ? Sound difficult. Anyways, if you are sure, let it be so.